lib/functions/plot_measures.R

```diff
@@ -29,62 +29,99 @@ plot_measures <- function(
     facet_wrap = FALSE,
     facet_var = NULL,
     colour_var = NULL,
+    shape_var = NULL,
+    save_path = NULL,
```

Comment:
I think we should not have the saving part of the plotting function call

Reply:
Will change this!